### DISCUSSION TITLE
Matrix 96x8

### DISCUSSION THREAD

**Discussion by Puh73** [pixelit-project/PixelIt#75] [Ideas]
Ist es möglich, die Matrix auf 96x8 Pixel zu vergrößern um drei 32x8 Module hintereinander zu hängen?

Danke! 

Gruß Patrick

**Comment by o0shojo0o** [comment]
Von aktuellen Code Design nicht, man müsste allerhand dafür ändern.
Daher muss ich aktuell leider nein sagen. 

Gruß

**Comment by figugegu** [comment]
Bin ich richtig, dass auch "nur" 2x32, also 8x64, nicht geht, oder? 
Danke und Gruss

**Reply by foorschtbar** [reply]
8 Pixel in der höhe und 32 in der breite ist fix.

**Reply by figugegu** [reply]
Danke für das rasche Feedback.
Gruss

**Reply by cheetahunter** [reply]
I have managed to connect 4 8x32 matrixes together

**Reply by DSRocker** [reply]
> I have managed to connect 4 8x32 matrixes together

Would you please share the Code
